Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -551,7 +551,20 @@ def install
 
   def test_home
     assert_equal HOMEBREW_WWW,
-                 cmd("home", {"HOMEBREW_BROWSER" => "echo"})
+                cmd("home", {"HOMEBREW_BROWSER" => "echo"})
+                 
```

Comment:
Some maintainers (including me) have either created or merged code with whitespace issues in the past, thus no need to be too embarrassed about this. We all make mistakes (and this is a tiny one). Thankfully there are also often tools/settings that help avoid them. 😸